### PR TITLE
{bio}[intel/2017a] VariantMetaCaller 1.0

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2677.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fe98d9a7b0b804f41d30648d6254e3c9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2003.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a50c50fb94676962fe2038ceb14bf61b for a full test report.

File: easybuild/easyconfigs/v/VariantMetaCaller/VariantMetaCaller-1.0_fix-hardcoding.patch [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+diff --git a/VariantMetaCaller_v1.0/makefile.orig b/VariantMetaCaller_v1.0/makefile
+index 78ff65d..c49027d 100644
+--- a/VariantMetaCaller_v1.0/makefile.orig
++++ b/VariantMetaCaller_v1.0/makefile
+@@ -37,7 +37,7 @@ all: VariantMetaCaller
+ VariantMetaCaller: $(OBJS) $(USER_OBJS)
+ 	@echo 'Building target: $@'
+ 	@echo 'Invoking: GCC C++ Linker'
+-	g++ -o"VariantMetaCaller" $(OBJS) $(USER_OBJS) $(LIBS) -fopenmp
++	${CXX} ${CXXFLAGS} -o"VariantMetaCaller" $(OBJS) $(USER_OBJS) $(LIBS) -fopenmp
```
Comment (wpoely86) [inline easybuild/easyconfigs/v/VariantMetaCaller/VariantMetaCaller-1.0_fix-hardcoding.patch]: `LDFLAGS` ?

Comment (wpoely86) [inline easybuild/easyconfigs/v/VariantMetaCaller/VariantMetaCaller-1.0_fix-hardcoding.patch]: Shouldn't `-fopenmp` come from EB?

Comment (boegel) [inline easybuild/easyconfigs/v/VariantMetaCaller/VariantMetaCaller-1.0_fix-hardcoding.patch]: I don't see the point, there are no dependencies?

Comment (wpoely86) [inline easybuild/easyconfigs/v/VariantMetaCaller/VariantMetaCaller-1.0_fix-hardcoding.patch]: it's the proper way of doing it. Why have `LIBS` ?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2677.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5318d97de8456f89be5e98e26703c22c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2003.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fa80f7dd00dabe0a123176ed45f105bc for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**